features/FEATURE_PSA/TARGET_TFM/src/os_wrapper_cmsis_rtos_v2.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017-2019, Arm Limited. All rights reserved.
+ * Copyright (c) 2017-2020, Arm Limited. All rights reserved.
```

Comment:
In commit messge "Change OS Wrapper layer", the title "Change OS Wrapper layer" isn't very precise. Consider saying something like "Allocate TF-M objects from the heap" instead.

Reply:
Fixed.